todo/views.py

```diff
@@ -0,0 +1,17 @@
+from .models import Todo
+from .serializers import TodoSerializer
+from rest_framework import mixins
+from rest_framework import generics
+
+class SnippetList(mixins.ListModelMixin,
+                  mixins.CreateModelMixin,
+                  mixins.DestroyModelMixin,
+                  generics.GenericAPIView):
```

Comment:
Since we are not modifying any default behavior, can we use a generic viewset instead of multiple mixins?